microsoft-edge/devtools-guide-chromium/dom/index.md

```diff
@@ -139,28 +139,54 @@ To edit attributes, double-click the attribute name or value.  Follow the instru
 
 1. Under **Edit Attributes**, right-click **Howard** and then select **Inspect**.
 
-1. Double-click `<li>`.  The text is highlighted to indicate that the node is selected.
+1. Double-click `<p>`.  The text is highlighted to indicate that the node is selected.
 
    ![Edit the node.](../media/dom-elements-highlighted-edit-attributes-highlighted.msft.png)
 
 1. Select the `Right` arrow key, add a space, type `style="background-color:gold"`, and then press `Enter`.  The background color of the node changes to gold.
 
    ![Add a style attribute to the node.](../media/dom-elements-highlighted-edit-attributes-inline-css.msft.png)
 
-### Edit node type
+### Edit node tag name
 
-To edit the type of a node, double-click the type and then type in the new type.<!--do the following steps do that?-->
+To edit the tag name of a node, double-click the tag name and then type in the new tag name.
 
 1. Open the [DOM Examples](https://microsoftedge.github.io/Demos/devtools-dom-get-started/) demo page in a new window or tab.
 
-1. Under **Edit Node Type**, right-click **Hank** and then select **Inspect**.
+1. Under **Edit Node Tag Name**, right-click **Hank** and then select **Inspect**.
 
-1. Double-click `<li>`.  The text `li` is highlighted.
+1. Double-click `<p>`.  The text `p` is highlighted.
 
-1. Delete `li`, type `button`, then press `Enter`.  The `<li>` node changes to a `<button>` node.
+1. Delete `p`, type `button`, then press `Enter`.  The `<p>` node changes to a `<button>` node.
 
    ![Change the node type to button.](../media/dom-elements-highlighted-edit-node-type-button.msft.png)
 
+### Edit multiple nodes, text, and attributes 
+
+To change the tag name, text content, or attributes of multiple nodes at once, you can edit part of the DOM using an HTML text editor.
+
+To use the HTML editor:
+
+1. Open the [DOM Examples](https://microsoftedge.github.io/Demos/devtools-dom-get-started/) demo page in a new window or tab.
+
+1. Right-click **Edit Content, Tag Name, and Attributes**, and then select **Inspect**.
+
+1. In the DOM Tree, right-click the `<section id="edit-as-html-1">` node and select **Edit as HTML**. The HTML editor appears.
+
+   ![The HTML editor.](../media/dom-elements-highlighted-edit-html-editor.png)
```

Comment:
Consider adding red box around webpage's section Edit Content, Tag Name, and Attributes, since UI steps are done there, and since this stumped me when spot-checking.

Reply:
ok